sanic/app.py

```diff
@@ -146,7 +152,7 @@ def register_listener(self, listener, event):
         :param event: when to register listener i.e. 'before_server_start'
         :return: listener
         """
-
+        subscribe(event_name=event, callback=listener, signals=self.signals)
```

Comment:
If self.listener() decorator calls `subscribe()` (on line 141 above) then why does this helper function also need to call `subscribe()`? Wouldn't this result in two subscribes with the same listener and signals?